schemas/smtp.py

```diff
@@ -0,0 +1,22 @@
+# zschema sub-schema for zgrab2's smtp module
+# Registers zgrab2-smtp globally, and smtp with the main zgrab2 schema.
+from zschema.leaves import *
+from zschema.compounds import *
+import zschema.registry
+
+import schemas.zcrypto as zcrypto
+import schemas.zgrab2 as zgrab2
+
+smtp_scan_response = SubRecord({
+    "result": SubRecord({
+        "banner": String(),
+        "ehlo": String(),
+        "smtp_help": String(),
```

Comment:
Seems like this could just be `help`. Should be apparent that it's for SMTP...

Reply:
I can get behind that. I originally left it there for symmetry with the input `—smtp-help` (since `—help` is taken), but that’s not a concern on the output.

Reply:
I figured that was the reason. I wondered if something like `--help-command` or `--send-help` might be less confusing than `--smtp-help`, since that might also sound like something that provides the user with help information.